src/sentry/integrations/gitlab/integration.py

```diff
@@ -106,7 +116,7 @@ class InstallationForm(forms.Form):
     group = forms.CharField(
         label=_("GitLab Group Name"),
         widget=forms.TextInput(
-            attrs={'placeholder': _('my-awesome-group')}
+            attrs={'placeholder': _('example-co')}
```

Comment:
I don't have a preference here, but could you tell me a bit about this change so I understand why it was made?

Reply:
I was trying to make the 'example' text more consistent. We have `http://gitlab.example.com` for the domain.

Reply:
oh gotcha. sounds fine. just curious :)